sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md

```diff
@@ -30,65 +31,22 @@ az keyvault create --resource-group <your-resource-group-name> --name <your-key-
 ### Authenticate the client
 
 In order to interact with the Azure Key Vault service, you'll need to create an instance of the [CertificateClient][certificate_client_class] class. You need a **vault url**, which you may see as "DNS Name" in the portal,
- and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object.
+and credentials to instantiate a client object.
 
-Client secret credential authentication is being used in this getting started section but you can find more ways to authenticate with [Azure identity][azure_identity]. To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below,
-or other credential providers provided with the Azure SDK, you should install the Azure.Identity package:
+The examples shown below use a [`DefaultAzureCredential`][DefaultAzureCredential], which is appropriate for most scenarios including local development and production environments.
+Additionally, we recommend using a managed identity for authentication in production environments.
+You can find more information on different ways of authenticating and their corresponding credential types in the [Azure Identity][azure_identity] documentation.
+
+To use the `DefaultAzureCredential` provider shown below,
+or other credential providers provided with the Azure SDK, you must first install the Azure.Identity package:
```

Comment:
To use the `DefaultAzureCredential` shown below, or other credential types provided with the Azure Identity library, you must first install the `Azure.Identity` package:

Reply:
Markdown elides new lines in consecutive lines when rendered.